### ISSUE TITLE
Countable items doesn't have serial

### ISSUE THREAD

**Issue by xGuTeK** [ko4life-net/ko#151]
#### Description
Countable items doesn't have serial so if someones find way to duplicate items i it will be undetectable. We need to secure that.

**Comment by stevewgr** [comment]
Good point. Also keeping in mind that we have an `ItemManager` project in our server files pack, which is supposed to act like a dupe scanner, verifying serials and such.
